### ISSUE TITLE
Article about fixing BrowserShot on MacOS + Valet

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/community-forum#377]
That way, we can actually test our website properly 😀

**Comment by tabacitu** [comment]
@karandatwani92 - our website is using https://github.com/spatie/browsershot, which in itself is using [Pupeteer](https://github.com/puppeteer/puppeteer). But both Mohammad and I have been unable to _properly_ install and use BrowserShot on our MacOS + Valet setups.  Everything works great on staging & production, but on localhost... it does NOT. The PDFs get exported blank (0 bytes).

I'd really like us to find a solution to this. We can shoot two birds with one stone here:
- we fix a problem in our workflow; it's not ok that some of us can't properly test every bit of functionality our website has;
- we create an article about how people can do the same; which is broad enough that we can post on laravel-news;

I've spent hours trying to get this to work. And I failed.
Mohammad has spent hours trying to get this to work. And he failed.

Maybe you'll have more luck than we did, Karan? Have you used BrowserShot before? You have MacOS + Valet too, correct?

**Comment by o15a3d4l11s2** [comment]
@tabacitu, I just set up BrowserShot on my Valet environment and it works properly, with the only addition being `ignoreHttpsErrors` like
```
Browsershot::url('https://eshop.test/page/about-us')->ignoreHttpsErrors()->save('example.pdf');
```

At a side note I installed puppeteer through npm, globally
```
npm install puppeteer --location=global
```

Is the website repository available, so I can set it up locally and test?

**Comment by karandatwani92** [comment]
Hello @tabacitu,
I had never used valet. But I tried, it was simple.
I tried Browsershot + Pupeteer along with Valet. It worked. 

Yes, we can write an article on this. 🎉

**Comment by tabacitu** [comment]
Ugh. Thanks for the feedback, guys. Knowing it's working for you... I hunkered down and spent the time to fix it on my machine. In fact, I've just spent more than 4 grueling hours today fixing it. But in the end... I've got it working.

I'll close this issue, as it seems the problem was MY installation and MY inability to work out what was going wrong. But for posterity, I've added my findings in the README file of the package I'm launching today - https://github.com/Laravel-Backpack/download-operation

Thanks again! False alarm.
